src/Microsoft.ML.Core/Data/IHostEnvironment.cs

```diff
@@ -68,11 +68,6 @@ public interface IHostEnvironment : IChannelProvider, IProgressChannelProvider
         /// </summary>
         int ConcurrencyFactor { get; }
 
-        /// <summary>
-        /// Flag which indicate should we stop any code execution in this host.
-        /// </summary>
-        bool IsCancelled { get; }
```

Comment:
>IsCancelled [](start = 13, length = 11)

I'm fine with removing this infra for now as nothing uses it.  We should consider, in the future to allow iterative components to be cancellable (Boosting, linear, etc)

Reply:
Agreed, I have updated the issue with the same.